packages/api/identity/src/route/identities.rs

```diff
@@ -172,7 +172,11 @@ pub async fn validate_profile(
 		"invalid parameter account_number`"
 	);
 
+<<<<<<< HEAD
```

Comment:
changed marker

Reply:
might've been github.dev glitch, marker is gone